v4_MIGRATION_GUIDE.md

```diff
@@ -286,7 +288,7 @@ The Google Pay feature is now supported in a single dependency:
 
 ```groovy
 dependencies {
-  implementation 'com.braintreepayments.api:google-pay:4.0.0-beta1'
+  implementation 'com.braintreepayments.api:google-payment:4.0.0-beta1' // we think this should probably be google-pay, but it's deployed as google-payment
```

Comment:
This was fixed in #335, so the next beta release should have the correct `google-pay` package name